server/bingo/tests.py

```diff
@@ -1,3 +1,21 @@
-from django.test import TestCase # noqa
+from django.test import TestCase
+from .models import Challenge
 
-# Create your tests here.
+
+class ChallengeTest(TestCase):
+    def setUp(self):
+        Challenge.objects.create(
+            name="Test",
+            description="This is a test challenge.",
+            challenge_type="act",
+            points=5,
+            total_completions=10,
+        )
+
+    def test_create_challenge(self):
+        challenge = Challenge.objects.get(name="Test")
+        self.assertEqual(challenge.description, "This is a test challenge.")
+        self.assertEqual(challenge.challenge_type, "act")
```

Comment:
Can you add a test which checks that there is an error if we try to create a challenge with ``challenge_type`` something other than the allowed values?